packages/plugin-link/src/components/LinkComponent.ts

```diff
@@ -0,0 +1,49 @@
+import { OwlComponent } from '../../../plugin-owl/src/ui/OwlComponent';
+import { InlineNode } from '../../../plugin-inline/src/InlineNode';
+import { LinkFormat } from '../LinkFormat';
+import { LinkParams } from '../Link';
+import { Layout } from '../../../plugin-layout/src/Layout';
+import { useState } from '@odoo/owl';
+
+export class LinkComponent<T = {}> extends OwlComponent<T> {
+    static components = {};
+    static template = 'link';
+    state = useState({
+        url: this._currentUrl(),
+        label: this._currentLabel(),
+    });
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    async saveLink(): Promise<void> {
+        await this.env.editor.execCommand('link', {
+            url: this.state.url,
+            label: this.state.label,
+        } as LinkParams);
+        await this.env.editor.plugins.get(Layout).remove('link');
```

Comment:
You're removing `link` in the plugin already and you're doing it here as well ? At least one of them should disappear. Honestly, I think both should disappear as calling `destroy` should remove it IMO. cc @Gorash